README.markdown

```diff
@@ -75,6 +76,119 @@ bin/rake prepare_translations
 before pushing to GitHub. This will normalize translation file formatting and
 check for missing or unused keys.
 
+### Conventions
+
+#### Views and Helpers
+
+In the view layer (templates and helper modules), externalize strings using the
+`I18n.t()` translation helper directly. Templates typically use the
+["lazy" lookup][i18n-lazy] form.
+
+To aid translation, it's desirable to externalize strings in a form as close to
```

Comment:
I think ideally, this is expounded upon, with examples, in a blog post which we can point to. I don't know that this belongs in the readme, or else it needs to be expanded.

It's pretty complicated to read and I _think_ I understand, because I have heard you talk about this previously, but I don't know if this advice is actionable.

Reply:
Perhaps it does belong in _a_ readme. Looking at this, it's a big chunk of text - and it's all useful information.

Would it be reasonable to put this in a separate localization document, which was linked to from the main readme?

Reply:
Makes sense — I'll extract it to a separate doc.